tests/test_constraints.py

```diff
@@ -1148,14 +1147,26 @@ async def test_constraints_ddl_04(self):
 
         # making sure the constraint was applied successfully
         async with self.assertRaisesRegexTx(
-            edgedb.ConstraintViolationError,
-            'foo must be no longer than 3 characters.',
+            edgedb.ConstraintViolationError, 'My custom message.'
         ):
-            await self.con.execute("""
-                INSERT ConstraintOnTest3 {
-                    foo := 'Test'
-                };
-            """)
+            try:
+                await self.con.execute(
+                    """
+                    INSERT ConstraintOnTest3 {
+                        foo := 'Test'
+                    };
+                    """
+                )
+            except edgedb.ConstraintViolationError as e:
+                # edgedb-python does not expose a nicer access to details field
+                details = e._attrs[2].decode("utf-8")
```

Comment:
Am I doing something wrong here? Or does edgedb-python really don't expose error details field?

Reply:
Turns out that edgedb-python really does not expose details.

Reply:
I'll wait for a release of edgedb-python and clean this up before merging.